src/os/state/basestatemanager.js

```diff
@@ -141,17 +141,77 @@ os.state.BaseStateManager.prototype.registerPersistable = function(key, clazz) {
 
 
 /**
- * Gets a registered {@link os.IPersistable} class.
+ * Gets an instance of a registered {@link os.IPersistable} class.
  *
  * @param {string} key The object key
  * @return {os.IPersistable}
+ * @suppress {checkTypes} To allow the catch() failover to exist
```

Comment:
Whatever this is suppressing, we should stop doing it and get rid of the statement. Suppress calls should never be used to try to help find an error as they're more likely to cause even more errors.

Reply:
Removed. Also removed the corresponding branching and logging types in the code.